android/app/src/main/res/layout/fragment_bookmark_list.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
   xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:mapsme="http://schemas.android.com/apk/app"
   xmlns:tools="http://schemas.android.com/tools"
   xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
What is the difference between /apk/app and /apk/res-auto ?

Reply:
I think /apk/app redirects to apk/res-auto because properties prefixed with mapsme are recognized with app namespace.
Maybe use /apk/app add ability to set custom properties in layout?